compiler/rustc_codegen_ssa/src/traits/backend.rs

```diff
@@ -54,6 +55,12 @@ pub trait CodegenBackend {
     fn print_passes(&self) {}
     fn print_version(&self) {}
 
+    /// If this plugin provides additional builtin targets, provide the one enabled by the options here.
```

Comment:
```suggestion
    /// If this codegen backend provides additional builtin targets, provide the one enabled by the options here.
```

The llvm codegen backend is not in any way a plugin. It is linked into `librustc_driver.so`.